workflow-examples/jobs-in-parallel/jobs_in_parallel.groovy

```diff
@@ -0,0 +1,10 @@
+def branches = [:]
+for (int i = 0; i < 4; i++) {
+  def which = "${i}" 
```

Comment:
As this repo is meant to be educational, I would say that pretty much every line in this script should have comments.  Especially non-obvious code like this.
The similar [parallel-for-list](https://github.com/jenkinsci/workflow-examples/blob/41b0312/workflow-examples/parallel-from-list/parallelFromList.groovy) script is a good example.

In any case, from that Stack Overflow answer:
> Beware that you should **not** be using a “dummy” parameter value to differentiate queue entries. 

So it sounds like this perhaps isn't a good idea for a Workflow example?